custom_components/tahoma/__init__.py

```diff
@@ -143,6 +146,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
                 device.widget,
             )
 
+        if device.widget == HOMEKIT_STACK:
```

Comment:
I would prefer to have the condition inside the method too.

Reply:
Sorry for merging in advance @tetienne, I will work on it in a future PR...